docs/usage.rst

```diff
@@ -0,0 +1,197 @@
+Naggum usage
+============
+
+Currently there are two dialects of Naggum: high level *Compiler* and low-level
+*Assembler*.
+
+Naggum Compiler
+---------------
+
+Command line syntax for Naggum Compiler is::
+
+    $ Naggum.Compiler [source1.naggum] [source2.naggum] [/r:assembly1] [/r:assembly2]
+
+Each input source file will be compiled to a separate executable assembly (i.e.
+an ``.exe`` file) in the current directory.
+
+``.naggum`` extension is recommended for high level Naggum files.
+
+Naggum Assembler
+----------------
+
+Naggum Assembler uses low level Naggum dialect. Command line syntax is::
+
+    $ Naggum.Assembler [source1.nga] [source2.nga]
+
+Assembler output is determined based on input file content: each file may
+contain multiple (or none) assembly constructs. Each assembly will be saved to
+its own executable file in the current directory.
+
+``.nga`` extension is recommended for low level Naggum files.
+
+S-expression syntax
+-------------------
+
+Each Naggum program (either high-level or low-level) is written as a sequence of
+S-expression forms. In s-expression, everything is either an atom or a list.
+Atoms are written as-is, lists should be taken into parens.
+
+Possible atom values are::
+
+    "A string"
+    1.4e-5 ; a number
+    System.Console ; a symbol
+
+A symbol may consist of any letter, digit or ``+-*/=<>!?.`` character sequence.
```

Comment:
Don't use "may" here; "symbol consists".

Moreover, I'd rewrite the whole sentence like that: "A symbol is a sequence consisting of letters, digits, and any of the following characters:``+-*/=<>!?.``".